packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -1,3 +1,19 @@
+/**
+* CSS Custom Properties  
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-handle-icon-color: Specifies the icon color of the component.
+* @prop --calcite-handle-icon-color-hover: Specifies the icon color of the component when in hover state.
```

Comment:
question: do we prefer the term `hover state` or `hovered`? cc @geospatialem 

Should this be worded like `defines the color of an icon sub-component when hovered  inside the component.`

Thats how i've seen these worded in other PRs. I guess we'll probably need to do some cleanup/consistency doc checks before we merge the "real" PR?

Reply:
Maybe we can check the epic PR every so often and fix any inconsistencies in wording/description?

Reply:
> question: do we prefer the term hover state or hovered? cc @geospatialem
> 
> Should this be worded like `defines the color of an icon sub-component when hovered  inside the component.`

Great question. The sentence reads a bit weird when coupled with `"when hovered"`. WDYT about another option altogether, and something like `"on hover"`? For instance:

> Specifies the component's icon color on hover.

Reply:
I like the simplicity of that phrasing.